sections/Chapter 3 Sprint 1/postgreSQL.tex

```diff
@@ -0,0 +1,41 @@
+\subsection{PostgreSQL}
+The group last year chose to implement the database using PostgreSQL which is an relational database system \cite{knox2020}.
+
+A relational database is a collection of tables, known as relations. Relational databases are used to provide access to related data points.
+These relations consist of rows, known as tuples. Each tuple represents an entity with specific attributes represented by the columns of the given relation.
+In the context of an enterprise company, one could for example model an employee and a manager, and the relationship between them.
+The goal of the designing such relations is to represent entities as intuitively as possible, making it easy to establish relationships between them. 
```

Comment:
Is that the only goal of designing relations? It also breaks a bit away from the description of relational databases as mathematical concepts - then its immediately returned to with a description of keys.